src/pages/faq.md

```diff
@@ -266,4 +266,4 @@ still an abbreviation) to desperately avoid yelling TUS in all caps. We used to
 insist on lowercasing, but have gotten more pragmatic from our encounters with
 journalists, who insist on it being Tus in their articles, and there's not much
 we can do about it. So now we just say: write it however you feel like that day
-:smile:
+😄
```

Comment:
Does this need to be wrapped in an `<Emoji>` tag?

Reply:
It doesn't need to be (but we can do it "just to be safe"). This emoji doesn't have a non-color icon equivalent in most fonts and will always show up as the color emoji defined in the font for the users' platform.

The ⚠️ emoji does have such a non-color equivalent, and so we force-set it to one of those color emoji system fonts.